### PR TITLE
docs: fix typos and couple spelling changes

### PR DESCRIPTION
**Pull request by hozefaj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,17 +50,17 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
...
 
 🔗 [**Read More: structure by components**](/sections/projectstructre/breakintcomponents.md)
 
 <br/><br/>
 
 ## ![✔] 1.2 Layer your components, keep Express within its boundaries
 
-**TL;DR:** Each component should contain 'layers' - a dedicated object for the web, logic and data access code. This not only draws a clean separation of concerns but also significantly eases mocking and testing the system. Though this is a very common pattern, API developers tend to mix layers by passing the web layer objects (Express req, res) to business logic and data layers - this makes your application dependant on and accessible by Express only
+**TL;DR:** Each component should contain 'layers' - a dedicated object for the web, logic, and data access code. This not only draws a clean separation of concerns but also significantly eases mocking and testing the system. Though this is a very common pattern, API developers tend to mix layers by passing the web layer objects (Express req, res) to business logic and data layers - this makes your application dependant on and accessible by Express only
```
Comment (BrunoScheufler) [inline README.md]: There's still another `dependant` in here, which would need to be fixed 😄

Comment (hozefaj) [reply README.md]: Done. Changes made.

**Comment by i0natan** [comment]
@BrunoScheufler Who's assigned? need other reviewers?

**Comment by BrunoScheufler** [comment]
@i0natan I'd like your final review on this, as it doesn't take a long time, you can also merge it 👍

**Comment by i0natan** [comment]
@BrunoScheufler LGTM, the ball is in your hand

**Comment by BrunoScheufler** [comment]
@i0natan could you review once more? Had to fix some merge conflicts 👍

**Comment by i0natan** [comment]
@BrunoScheufler Sorry, missed this one, approved

@hozefaj Looks great 💯

**Pull request merged**